lib/test/reporter/dot.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+module.exports = async function*(source) {
```

Comment:
Can we name the function so we get clearer stack traces in case of errors?
```suggestion
module.exports = async function* dot(source) {
```